clang/test/SemaSYCL/loop_fusion.cpp

```diff
@@ -4,8 +4,8 @@
 
 [[intel::loop_fuse("foo")]] void func() {} // expected-error{{'loop_fuse' attribute requires an integer constant}}
 
-[[intel::loop_fuse(1048577)]] void func1() {}        // expected-error{{'loop_fuse' attribute requires integer constant between 0 and 1048576 inclusive}}
```

Comment:
Previously we gave error on the invalid range, now it is Ok. 

Is it expected that we will not give error any more for invalid attribute arguments?

If not, I think we can add like this:

if (ArgInt > 1048576) {
        Diag(E->getBeginLoc(), diag::err_attribute_argument_out_of_range)
            << CI.getAttrName() << 0 << 1048576<< E->getSourceRange();
        return;
      }
    }

Reply:
This is expected behavior. We are removing the upper bounds for some attributes since it was arbitrary ( #2911).

Reply:
Thanks for the clarification. Sorry I misunderstood.